### ISSUE TITLE
test_normalized_laplacian test fails occasionally

### ISSUE THREAD

**Issue by huonw** [stellargraph/stellargraph#542]
**Describe the bug**

Sometimes 

https://github.com/stellargraph/stellargraph/blob/ef5588038896c1a65db467a768f2e023c6562611/tests/core/test_utils.py#L72-L81

fails on CI. I haven't investigated reproducing locally.

Examples:

- https://buildkite.com/stellar/stellargraph/builds/1862#277ab375-5ff3-4ce9-831f-8df3d7cc8da8
- https://buildkite.com/stellar/stellargraph/builds/1845#fd9131cb-0e8f-43ba-ade4-d5dc00212ace

Example output:

```
__________________________ test_normalized_laplacian ___________________________
    def test_normalized_laplacian():
        node_list = list(pytest.G.nodes())
        Aadj = nx.adjacency_matrix(pytest.G, nodelist=node_list)
        laplacian = normalized_laplacian(Aadj)
>       assert 1 == pytest.approx(laplacian.sum(), 0.2)
E       AssertionError: assert 1 == 1.3477525659143204 ± 2.7e-01
E        +  where 1.3477525659143204 ± 2.7e-01 = <function approx at 0x7f1a043607a0>(1.3477525659143204, 0.2)
E        +    where <function approx at 0x7f1a043607a0> = pytest.approx
E        +    and   1.3477525659143204 = <bound method _cs_matrix.sum of <6x6 sparse matrix of type '<class 'numpy.float64'>'\n	with 18 stored elements in Compressed Sparse Row format>>()
E        +      where <bound method _cs_matrix.sum of <6x6 sparse matrix of type '<class 'numpy.float64'>'\n	with 18 stored elements in Compressed Sparse Row format>> = <6x6 sparse matrix of type '<class 'numpy.float64'>'\n	with 18 stored elements in Compressed Sparse Row format>.sum
tests/core/test_utils.py:76: AssertionError
```

**To Reproduce**
1. Run the test on CI many times?

**Expected behavior**
The test shouldn't fail occasionally: either all the time or never.
**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Data used**
N/A

**Additional context**
N/A

**Comment by kieranricardo** [comment]
@huonw there seems to be a bug in the unit test. 

`normalize_adj(Aadj, symmetric=False).sum()` should equal the number of isolated nodes (1 in this case)

But `normalize_adj(Aadj, symmetric=True).sum()` should not. I'll make a PR removing this assertion for now and try to replace it with something meaningful.

**Comment by kieranricardo** [comment]
fixed with PR #569
